src/blurb/blurb.py

```diff
@@ -742,6 +742,15 @@ def get_subcommand(subcommand):
 
 
 
+@subcommand
+def version():
+    """
+Print blurb version.
+    """
```

Comment:
Why the weird indentation?  Is it just for consistency with the rest?

I would just use a oneliner anyway:
```suggestion
    """Print blurb version."""
```

Reply:
Yeah, consistency. Oneliner looks good.